solutions/devsprint-bruno-faganello-2/SpaceApp/Components/TabBar/SPTabBar.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  SPTabBar.swift
+//  SpaceApp
+//
+//  Created by Paolo Prodossimo Lopes on 25/04/22.
+//
+
+import SwiftUI
+
+fileprivate typealias Constants = SPConstants.SPTabBar
+
+struct SPTabBar: View {
+    //MARK: - Properties
+    @State private var currentPage: SPTabBarPages = .home
```

Comment:
[Question]
Why not use TabView and tabItem and make some custom.
I think It's gonna be better

Reply:
I don't know this component existed (I had looked for it waiting for the compiler to help and it didn't show up) kkk